File: module22/module01/module43/src/main/java/com/android/example/module22_module01_module43/ClassAAC.java
package com.android.example.module22_module01_module43;

public class ClassAAC {
	private 	com.android.example.module01_module39.ClassAAJ instance_var_1_0 = new com.android.example.module01_module39.ClassAAJ();
	private 	com.android.example.module01_module39.ClassAAG instance_var_1_1 = new com.android.example.module01_module39.ClassAAG();
	private 	com.android.example.module01_module39.ClassAAF instance_var_1_2 = new com.android.example.module01_module39.ClassAAF();

	public void method0(
		com.android.example.module01_module39.ClassAAC param0,
		com.android.example.module01_module39.ClassAAA param1,
		com.android.example.module01_module39.ClassAAA param2) throws Throwable {
		if (new java.lang.Object().equals(new java.lang.Object())) {
			param0.method2(new io.reactivex.subscribers.TestSubscriber(), new io.reactivex.subscribers.TestSubscriber(), new io.reactivex.subscribers.TestSubscriber(), new io.reactivex.observers.TestObserver());

		} else {
			io.reactivex.subscribers.TestSubscriber<Object> local_var_3_0 = new io.reactivex.subscribers.TestSubscriber();
			local_var_3_0.hasSubscription();

		}
		io.reactivex.disposables.SerialDisposable local_var_2_3 = new io.reactivex.disposables.SerialDisposable();
		local_var_2_3.get();

		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_4 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_4.assertNotSubscribed();

	}

	public void method1(
		com.android.example.module01_module39.ClassAAA param0) throws Throwable {
		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_1 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_1.requestMore(811129842l);

		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_2 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_2.isDisposed();

	}
}

File: module06/module353/module3/src/main/java/com/android/example/module06_module353_module3/ClassAAC.kt
package com.android.example.module06_module353_module3

@Suppress("UNUSED_PARAMETER")
class ClassAAC {
	private 	val instance_var_1_0: rx.observers.TestSubscriber<kotlin.Any> = rx.observers.TestSubscriber()
	private 	val instance_var_1_1: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()

	fun method0(
		param0: com.android.example.module06_module353_module2.ClassAAH,
		param1: io.reactivex.subscribers.TestSubscriber<kotlin.Any>,
		param2: io.reactivex.subscribers.TestSubscriber<kotlin.Any>,
		param3: rx.observers.TestSubscriber<kotlin.Any>
	) {
		val local_var_2_4: com.android.example.module06_module353_module2.ClassAAF = com.android.example.module06_module353_module2.ClassAAF()
		local_var_2_4.method1("SomeString")

		if (kotlin.Any().equals(kotlin.Any())) {
			val local_var_3_0: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
			local_var_3_0.cancel()

		} else {
			val local_var_3_0: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
			local_var_3_0.isDisposed()

		}
		param0.method0("SomeString", "SomeString", "SomeString")

	}

	fun method1(
		param0: io.reactivex.subscribers.TestSubscriber<kotlin.Any>,
		param1: io.reactivex.internal.util.LinkedArrayList
	) {
	}

	fun method2(
		param0: com.android.example.module06_module353_module2.ClassAAA,
		param1: com.android.example.module06_module353_module2.ClassAAE,
		param2: io.reactivex.subscribers.TestSubscriber<kotlin.Any>,
		param3: io.reactivex.subscribers.TestSubscriber<kotlin.Any>
	) {
		val local_var_2_4: rx.observers.TestSubscriber<kotlin.Any> = rx.observers.TestSubscriber()
		local_var_2_4.onError(kotlin.Throwable())

		val local_var_2_5: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_5.isDisposed()

		val local_var_2_6: com.android.example.module06_module353_module2.ClassAAH = com.android.example.module06_module353_module2.ClassAAH()
		local_var_2_6.method1("SomeString", "SomeString", "SomeString")

		param0.method0("SomeString", "SomeString", "SomeString", "SomeString")

	}

	fun method3(
		param0: io.reactivex.subscribers.TestSubscriber<kotlin.Any>
	) {
		val local_var_2_1: com.android.example.module06_module353_module2.ClassAAD = com.android.example.module06_module353_module2.ClassAAD()
		local_var_2_1.method2("SomeString", "SomeString", "SomeString", "SomeString")

		val local_var_2_2: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_2.hasSubscription()

		val local_var_2_3: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_3.onError(kotlin.Throwable())

		val local_var_2_4: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_4.assertNotSubscribed()

	}
}
